19/slim/Dockerfile

```diff
@@ -1,13 +1,13 @@
 FROM debian:jessie
 
-ENV OTP_VERSION="19.0.7"
+ENV OTP_VERSION="19.1"
 
 # We'll install the build dependencies, and purge them on the last step to make
 # sure our final image contains only what we've just built:
 RUN set -xe \
-	&& OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-${OTP_VERSION}.tar.gz" \
-	&& OTP_DOWNLOAD_SHA256="584e08c7d810204163247dd2187bfc44175bac60d3e2577a0ea55a6a898c8dce" \
-	&& runtimeDeps=' \
+  && OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-${OTP_VERSION}.tar.gz" \
+  && OTP_DOWNLOAD_SHA256="caf320c07bdd4c6e11831a0b0d25645a29112007077dbf11eec22437f8b041ed" \
+  && runtimeDeps=' \
```

Comment:
this whole set looks clean, but could you not to change space format? it was a single tab